### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by scenaristeur**
some fix in examples

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ should prompt for a password.
 **logout()**
 
 ```javascript
-fileClient.logout().then( console.log( `Bye now!` )
+fileClient.logout().then( console.log( `Bye now!` ))
```
Comment (Otto-AA) [inline README.md]: If I'm not mistaken, it should be `fileClient.logout().then(() => console.log('Bye now!'))`.

Only passing `console.log('Bye now!')` passes the response of the console.log call as parameter, not a function which calls console.log

**Comment by jeff-zucker** [comment]
Both these points are valid.  The login part of the readme also needs to be modifed to show it returns session from which session.webId may be derived.  @bourgeoa is now co-maintainer and may be able to get to this sooner than I will.  In any case, thanks @scenaristeur and @Otto-AA for your sharp eyes and suggestions.

**Comment by jeff-zucker** [comment]
There was a typo in the example, if you downloaded before now, download again.  The correct flag to get the ACL files copied is fetcher.recursiveCopy( url, {copyACL:true})

**Comment by jeff-zucker** [comment]
This is no longer relevant to v1.0.0. because we no longer provide session methods.

**Pull request closed without merging**